tools/PI/DevHome.PI/Helpers/ETWHelper.cs

```diff
@@ -0,0 +1,168 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Diagnostics;
+using System.Globalization;
+using System.Linq;
+using System.Security.Principal;
+using DevHome.PI.Models;
+using Microsoft.Diagnostics.Tracing;
+using Microsoft.Diagnostics.Tracing.Session;
+using Serilog;
+
+namespace DevHome.PI.Helpers;
+
+internal sealed class ETWHelper : IDisposable
+{
+    private static readonly ILogger _log = Log.ForContext("SourceContext", nameof(ETWHelper));
+    private readonly Process targetProcess;
+    private readonly ObservableCollection<WinLogsEntry> output;
+
+    // TODO Add providers

```

Comment:
nit: can probably be removed. or converted to a comment about what this list is actually used for.

Reply:
sounds good, we already have a workitem tracking adding more providers or making it user customizable.